packages/block-library/src/classic/edit.js

```diff
@@ -84,12 +92,29 @@ export default class ClassicEdit extends Component {
 		}
 
 		editor.on( 'blur', () => {
+			editor.wpClassicBlockBookmark = editor.selection.getBookmark();
+
+			// Keep the "raw" content as it contains the caret position bookmark.
 			setAttributes( {
+				contentRaw: editor.getContent( { format: 'raw' } ),
```

Comment:
Storing the raw content without any processing seems like a bad idea? Why should this be different from the classic editor?